src/styles/theme/themes/light.ts

```diff
@@ -76,6 +76,7 @@ const lightTheme = {
     badgeAdHoc: colors.pink600,
     badgeAdHocHover: colors.pink700,
     mentionText: colors.blue600,
+    onboardingVideoBG: colors.green200,
```

Comment:
Can you share a screenshot of what this looks like in practice? We don't often use green200 so just curious how it turned out.

Reply:
This was a leftover of some early implementations for onboarding, thanks for pointing it out. It wasn't used anywhere, I removed it.